flow/table.go

```diff
@@ -257,6 +267,10 @@ func (ft *Table) update(updateFrom, updateTime int64) {
 			if f.FinishType != FlowFinishType_NOT_FINISHED {
 				delete(ft.table, k)
 			}
+		} else if now.Sub(time.Unix(0, f.Last * int64(time.Millisecond))) > ft.appTimeout[strings.ToLower(f.Application)] && ft.appTimeout[strings.ToLower(f.Application)] > 0 {
```

Comment:
See my upper comment, if you set the map with upper case application no need to call ToLower here